.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+## Pull Request checklist:
+
+- [ ] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
+- [ ] Create automatic tests
+- [ ] No automatic tests failures
+- [ ] Test manually the implemented changes
+- [ ] Review my own code (indentation, syntax, style, simplicity, readability)
```

Comment:
```suggestion
- [ ] Review your code (indentation, syntax, style, simplicity, readability)
```

IMO using "you" makes the sentence clearer and closer to the following sentence.